gammapy/spectrum/cosmic_ray.py

```diff
@@ -55,17 +33,53 @@ def cosmic_ray_flux(energy, particle="proton"):
     flux : `~astropy.units.Quantity`
         Cosmic ray flux in unit ``m^-2 s^-1 TeV^-1 sr^-1``
     """
-    pars = {
-        "electron": {"N": 6.85e-5, "k": 3.21, "L": 3.19e-3, "E_p": 0.107, "w": 0.776},
-        "proton": {"N": 0.096, "k": 2.70},
-        "N": {"N": 0.0719, "k": 2.64},
-        "Si": {"N": 0.0284, "k": 2.66},
-        "Fe": {"N": 0.0134, "k": 2.63},
+
+    proton = {
```

Comment:
Suggest to put the parameters directly where you create the model, to have model class and params in one place instead of two. Like this:
```
PowerLaw( 
   ...:    norm='1e9 cm', 
   ...:    index='asdf' 
   ...: )  
```